.github/workflows/docs_tags.yaml

```diff
@@ -12,13 +12,13 @@ jobs:
 
     steps:
     - name: Checkout
-      uses: actions/checkout@v3.5.2
```

Comment:
I'd like to double check why we specified a this version here. Maybe @danceratopz you know why?

Reply:
Double checked! Seems okay